ood-portal-generator/lib/ood_portal_generator/application.rb

```diff
@@ -160,11 +164,14 @@ def apache_changed_output
       def generate()
         view = View.new(context)
         dex = Dex.new(context, view, insecure)
-        rendered_template = view.render(template.read)
-        output.write(rendered_template)
-        if Dex.installed? && dex.enabled?
-          dex_output.write(dex.render)
-        end
+        dex_enabled = Dex.installed? && dex.enabled?
+
+        # convienence defaults for folks who are using dex or OIDC
+        view.auth = Dex.default_auth if dex_enabled && !view.auth?
+        content = view.auth? ? view.render(template.read) : view.render(no_auth_template.read)
```

Comment:
I'm fine breaking -t functionality but see other comment about using just a single template.

Reply:
Since `-t` is now broken under some conditions, can we just remove that flag?  Fine to defer with issue, just figured probably cleaner to have no `-t` than partially working `-t`.

Reply:
#1993. I've never used it, so I imagine it was only useful in the beginning when things were still a bit in progress.